controllers/aws/s3-bucket.go

```diff
@@ -158,6 +163,23 @@ func (a *AwsClient) findIfDiffTags(tagsToUpdate map[string]string, tagsFromAws [
 	a.Log.V(1).Info("returend from find diff Tags", "isDiffTags", isDiffTags, "newTags", newTags)
 	return isDiffTags, newTags
 }
+func (a *AwsClient) checkIfOwnerBucketByTag(bucketName string) (bool, error) {
```

Comment:
as the name states (maybe better to call this method `isBucketManagedByOperator`) i would expect this method to return bool value (true or false) and the error value should only return in case of technical error like timeout to AWS. the caller of this method should decide what to do with this true/false result. here you return true without error of false with error. the error has no meaning at this context